src/modules/Events/Misc/AutoContext.js

```diff
@@ -1,7 +1,13 @@
 const { Listener } = require('axoncore');
 
-const LOG_CATEGORIES = ['372085914765099008', '372088029495689226']; // Moderation and Logs
-const UPPER_STAFF_CATEGORIES = ['828540781291241492', '719883529470738523']; // Garden Gate and Lake Laogai
```

Comment:
Oh I just wanted to have Naga send the message in other staff channels rather than only working in dai-li or bot-interface. Let's still block it from sending admin channel links in other channels though (not that it happens anyways).

Reply:
Alright, sounds good. I'll get to it either this evening or tomorrow.